frontend/packages/client/src/components/ProposalCreate/StepThree.js

```diff
@@ -28,22 +27,16 @@ const StepThree = ({ stepsData, setStepValid }) => {
       })),
   };
 
-  const rawContentState = convertToRaw(
-    stepsData[0]?.description?.getCurrentContent()
-  );
-
-  const markup = draftToHtml(rawContentState);
+  const currentContent = stepsData[0]?.description?.getCurrentContent();
 
-  const htmlBody = markup
-    .replace(/target="_self"/g, 'target="_blank" rel="noopener noreferrer"')
-    .replace(/(?:\r\n|\r|\n)/g, "<br>");
```

Comment:
remove and handled by https://github.com/DapperCollectives/CAST/pull/31/files#diff-36c9fb6b191cd8f048fcd99f91b906996a9db94bc2b63602e2c7452209af6190R122-R135